### PR TITLE
reinforcement nukies and borgs get droppodded

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

reinforcements now get droppodded instead of appearing out of nowhere
fixes contractor drop pods having the wrong icon

## Why It's Good For The Game

it looks cool

## Changelog
:cl:
tweak: reinforcement nukies and borgs get droppodded into their location
fix: fixes contractor drop pods having the wrong icon
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
a man after my own heart

File: code/modules/antagonists/_common/antag_spawner.dm [hunk 1]
```diff
@@ -129,17 +138,18 @@
 			return
 		used = TRUE
 		var/mob/dead/observer/G = pick(nuke_candidates)
-		spawn_antag(G.client, get_turf(src), "syndieborg", user.mind)
+		spawn_antag(G.client, get_turf(src), "nukeop", user.mind)
```
Comment (ATH1909) [inline code/modules/antagonists/_common/antag_spawner.dm:141]: 👀

File: code/modules/antagonists/_common/antag_spawner.dm [hunk 2]
```diff
@@ -129,17 +138,18 @@
...
 		do_sparks(4, TRUE, src)
 		qdel(src)
 	else
 		to_chat(user, "<span class='warning'>Unable to connect to Syndicate command. Please wait and try again later or use the teleporter on your uplink to get your points refunded.</span>")
```
Comment (ATH1909) [inline code/modules/antagonists/_common/antag_spawner.dm:145]: this isn't a teleporter anymore

Comment (Fikou) [reply code/modules/antagonists/_common/antag_spawner.dm:145]: ok

File: code/modules/antagonists/_common/antag_spawner.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/antagonists/_common/antag_spawner.dm:141]: ok

**Comment by LemonInTheDark** [comment]
I like this. A lot

**Pull request merged**